backends/redis.go

```diff
@@ -55,99 +53,80 @@ func (e KeyError) Error() string {
 	return "Key not found"
 }
 
-func (r *RedisBackend) newRedisClusterPool() redis.UniversalClient {
+func (r *RedisBackend) Connect() (temporal.KeyValue, error) {
 	redisLogger.Info("Creating new Redis connection pool")
 
-	timeout := 5 * time.Second
-
-	if r.config.Timeout > 0 {
-		timeout = time.Duration(r.config.Timeout) * time.Second
-	}
-
-	var tlsConfig *tls.Config
-	if r.config.UseSSL {
-		tlsConfig = &tls.Config{
-			InsecureSkipVerify: r.config.SSLInsecureSkipVerify,
-		}
-	}
-
-	var client redis.UniversalClient
-	opts := &redis.UniversalOptions{
-		MasterName:       r.config.MasterName,
-		SentinelPassword: r.config.SentinelPassword,
-		Addrs:            r.getRedisAddrs(),
-		DB:               r.config.Database,
-		Username:         r.config.Username,
-		Password:         r.config.Password,
-		IdleTimeout:      240 * time.Second,
-		ReadTimeout:      timeout,
-		WriteTimeout:     timeout,
-		DialTimeout:      timeout,
-		TLSConfig:        tlsConfig,
+	conf := r.config
+	// ToDo: why we do not have TLS conf?
```

Comment:
We do have TLS Conf. It's not within `model.RedisOptions`

PS: Remove this comment :)